docs/team/account-members.md

```diff
@@ -0,0 +1,30 @@
+title: Sematext Account Members
+description: Inviting team members to your account means they get access to all your Apps, dashboards, notification hooks, alert rules, and everything else!  
+
+By inviting a user to your Account, you invite them to your **whole account**, 
+so they get access to **all** your Apps, Dashboards, notification
+hooks, alert rules, and integrations.
```

Comment:
Would it make sense to hyperlink some of these things?
Apps
Dashboards
Notification Hooks
Alerts
Integrations

Reply:
Yes. I will link it.